pkg/updatechecker/updatechecker.go

```diff
@@ -246,6 +247,23 @@ func checkForHelmAppUpdates(opts CheckForUpdatesOpts, finishedChan chan<- error)
 
 	availableUpdateTags, err := helm.CheckForUpdates(helmApp, license, &currentVersion)
 	if err != nil {
+		expiredErr := util.ActionableError{
+			NoRetry: true,
+			Message: "License is expired",
+		}
+		isExpired := false
+
+		newLicense, _ := helm.GetChartLicenseFromSecretOrDownload(helmApp)
```

Comment:
why not check for errors?

Reply:
if we can't sync and local copy is expired, what else is there to do....